src/patternfly/components/Badge/badge.hbs

```diff
@@ -1,3 +1,7 @@
-<span class="pf-c-badge {{ pf-c-badge--modifier }}">
+<span class="pf-c-badge {{#if badge--modifier}} {{badge--modifier}}{{/if}}"
+  {{#if badge--attribute}}
+    {{{ badge--attribute}}}
+  {{/if}}
+>
```

Comment:
move this after end of if

Reply:
can you clarify what I should be moving :)